src/main/java/uk/gov/hmcts/reform/civil/service/JudgementService.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.enums.MultiPartyScenario;
+import uk.gov.hmcts.reform.civil.enums.YesOrNo;
+import uk.gov.hmcts.reform.civil.model.CCJPaymentDetails;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.utils.MonetaryConversions;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.math.BigDecimal.ZERO;
+import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.getMultiPartyScenario;
+
+@Service
+@RequiredArgsConstructor
+public class JudgementService {
+
+    private static final String JUDGEMENT_BY_COURT = "The Judgement request will be reviewed by the court,"
+        + " this case will proceed offline, you will receive any further updates by post.";
+    private static final String JUDGEMENT_ORDER = "The judgment will order the defendant to pay £%s , including the claim fee and interest, if applicable, as shown:";
+    private final FeatureToggleService featureToggleService;
+
+    public CCJPaymentDetails buildJudgmentAmountSummaryDetails(CaseData caseData) {
+        BigDecimal claimAmount = caseData.getTotalClaimAmount();
+        MultiPartyScenario multiPartyScenario = getMultiPartyScenario(caseData);
+
+        if (caseData.isAcceptDefendantPaymentPlanForPartAdmitYes()) {
+            claimAmount = caseData.getRespondToAdmittedClaimOwingAmountPounds();
+        }
+        BigDecimal claimFee =  MonetaryConversions.penniesToPounds(caseData.getClaimFee().getCalculatedAmountInPence());
```

Comment:
To improve readability here, move these lines out to well-named private methods, then you can just call those directly from the builder method calls e.g. 
`.ccjJudgmentAmountClaimFee(getClaimFeeInPounds())` etc